shared/cluster/src/main/java/io/pravega/common/cluster/Cluster.java

```diff
@@ -60,4 +60,12 @@
      */
     public Set<Host> getClusterMembers();
 
+    /**
+     * Get the health status.
+     *
+     * @return true by default.
+     */
+    default public boolean isHealthy() {
```

Comment:
Again, lets have an abstract method instead of "default".

Reply:
Done